internal/pkg/template/jsonnet/function/function.go

```diff
@@ -124,6 +126,18 @@ func InstanceIDShort(instanceIDShort string) *jsonnet.NativeFunction {
 	}
 }
 
+// RandomUID Jsonnet function returns a random, shortened id of the template instance.
+func RandomUID() *jsonnet.NativeFunction {
```

Comment:
My bad, this should be `UUID`, `UID` refers to User Identifier

Reply:
https://github.com/keboola/keboola-as-code/pull/2162/commits/9f490950c46541b0e0cab2eb2b304be432e9ac31